app/javascript/listings/components/Modal.jsx

```diff
@@ -17,41 +17,13 @@ export const Modal = ({
 }) => {
   const shouldRenderMessageModal = listing && listing.contact_via_connect;
 
-  const Icon = () => (
-    <svg
-      width="24"
-      height="24"
-      viewBox="0 0 24 24"
-      className="crayons-icon pointer-events-none"
-      xmlns="http://www.w3.org/2000/svg"
-    >
-      <path d="M12 10.586l4.95-4.95 1.414 1.414-4.95 4.95 4.95 4.95-1.414 1.414-4.95-4.95-4.95 4.95-1.414-1.414 4.95-4.95-4.95-4.95L7.05 5.636l4.95 4.95z" />
-    </svg>
-  );
-
-  // TODO: Why are we not using the crayons modal component and instead recreating it here?
   return (
-    <div
-      role="dialog"
-      aria-modal="true"
-      aria-label="listing modal"
-      id="single-listing-container__inner"
-      className="single-listing-container__inner crayons-modal__box"
-    >
-      <div className="crayons-modal__box__header flex s:hidden">
-        <Button
-          type="button"
-          id="close-listing-modal"
-          tagName="button"
-          contentType="icon"
-          variant="ghost"
-          className="ml-auto"
-          icon={Icon}
-          onClick={onClick}
-          aria-label="Close listing"
-        />
-      </div>
-      <div className="crayons-modal__box__body p-0">
+    <div className="listings-modal">
```

Comment:
Tests (blocking): Add a test to ensure a listing loads with the modal refactor. It will require adding a listing into the E2E seed data. See seeds.rb to grab the script to seed it.

https://github.com/forem/forem/blob/fa3ba4d7d00a5a6beafd9b73522b6b26c0a88a6f/db/seeds.rb#L454-L524